src/mlpack/methods/ann/layer/lstm_impl.hpp

```diff
@@ -167,7 +167,10 @@ void LSTM<InputDataType, OutputDataType>::Forward(
 {
   //! Locally-stored cellState.
   OutputType cellState;
-  Forward(std::move(input), std::move(output), std::move(cellState), false);
+  // Trying a copy here as done in other layers rather than using std::move.
+  inputTemp = arma::mat(input.memptr(),
```

Comment:
I don't think this is required.

Reply:
You mean `inputTemp`?

Reply:
Yes, I meant inputTemp, I think `std::move` should work fine.